pkgs/by-name/th/thc-secure-delete/package.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "thc-secure-delete";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gordrs";
+    repo = "thc-secure-delete";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-hvWuxFkzhOSCplPtyjRtn36bIk6KdPBcpr3lAmiAyfE=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  makeFlags = [
+    "CC=${stdenv.cc.targetPrefix}cc"
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 srm $out/bin
+    install -Dm755 sfill $out/bin
+    install -Dm755 sswap $out/bin
+    install -Dm755 sdmem $out/bin
+    install -Dm755 sfill.1 $out/share/man/man1/sfill.1.gz
+    install -Dm755 srm.1 $out/share/man/man1/srm.1.gz
+    install -Dm755 smem.1 $out/share/man/man1/smem.1.gz
+    install -Dm755 sswap.1 $out/share/man/man1/sswap.1.gz
```

Right now this copies the binaries so that $out/bin is the executable directly, also these commands can be shorter and man pages should probably not have the x bit

```
$ results/thc-secure-delete/bin --help
results/thc-secure-delete/bin: illegal option -- -
smem v3.1 (c) 1997-2003 by van Hauser / THC <vh@thc.org>
```

```suggestion
    install -d $out/bin $out/share/man/man1
    install -m755 srm sfill sswap sdmem $out/bin
    install -m644 sfill.1 srm.1 smem.1 sswap.1 $out/share/man/man1
```

---

can you check the man pages again this time.

---

It looks fine now even though I'm not exactly sure how :^) You're not adding the .gz ending explicitly anymore, I assume it gets added by the standard mkDerivation builder or something